Makefile

```
@@ -0,0 +1,169 @@
+	@echo "  make shell          Open a shell inside the running container"
+	@echo "  make clean          Remove containers and networks"
+	@echo "  make config         Scaffold runtime directory and sample config"
+	@echo "  make edit           Edit runtime config file"
+	@echo "  make use-prebuilt   Use prebuilt GHCR image"
+	@echo "  make use-source     Enable local source builds via compose override"
+	@echo "  make update-compose Refresh local compose files from sample templates"
+	@echo ""
+	@echo "Runtime directory on host: $(BIBLEBOT_HOST_HOME)"
+	@echo "Container runtime directory: /data (BIBLEBOT_HOME=/data)"
+
+config:
```

checkmake (0.3.2)

[warning] Target body for "config" exceeds allowed length of 5 lines (13).

(maxbodylength)